### ISSUE TITLE
Allow "Finished Actor Run" trigger to GET more than 100 records

### ISSUE THREAD

**Issue by drobnikj** [apify/apify-zapier-integration#31]
**From reported issue in Zapier:**

Hi there, Apify Team!

JC here, reaching out from Zapier Support. One or more of our mutual customers has requested a new feature in your app.

Specifically, they'd like to see the Finished Actor Run trigger to GET more than 100 records, and not be limited to just 100.

An example use case for this is when a user has a "paid player" on Apify i.e. Tiktok Scraper that allows them to retrieve data from Tiktok but the scaper returns more than 100 records. All exceeding records are lost and are not sent to their downstream app, so removing this limit will allow them to do this.

Here's a GIST where we can see the endpoint query has a 100 record limit:

https://cdn.zapier.com/storage/files/c5c95c85dd8b0e9c548bd36a603a96ea_2.txt

Thanks, and please let us know if you have any questions!

cc https://developer.zapier.com/app/15018/issues/11

**Comment by drobnikj** [comment]
**I answered on the issue:**

Hi there,

Jakub is here from Apify team.

The data from the actor are stored in Apify dataset. We only attached part of the data in trigger data, as there can be a large number of items. But for some use cases, the 100 items are enough.

I would recommend using one more Apify action called "Fetch Dataset Items in Apify" to fetch the actor's data using this step in the Zap. In this action user can set the limit for how many items from the dataset need so that he can get the whole data. Users can find the dataset ID under the field "Default dataset ID" in the previous step trigger "Finished Actor Run".

I hope it helps.

Have a nice day, Jakub Drobník
